packages/compass-workspaces/src/index.ts

```diff
@@ -164,6 +164,21 @@ export function activateWorkspacePlugin(
     }
   });
 
+  // TODO(COMPASS-8033): activate this code and account for it in e2e tests and
```

Comment:
Splitting this out to a separate ticket because it requires some extra researchi into how this will affect our tests and generally electron environment where the browser warning is not shown by default